cmd/awl-tray/os_hacks_linux.go

```diff
@@ -49,22 +51,32 @@ func initOSSpecificHacks() {
 	}
 	nonRootUid = uid
 	config.LinuxFilesOwnerUID = uid
-	logRootHack("found uid under root: %d", uid)
+	logHack("found uid under root: %d", uid)
 
-	logRootHack("start connectSessionDbus")
+	logHack("start connectSessionDbus")
 	connectSessionDbus(nonRootUid)
-	logRootHack("end connectSessionDbus")
+	logHack("end connectSessionDbus")
 
 	display := os.Getenv("DISPLAY")
-	logRootHack("got DISPLAY from env: '%s'", display)
+	logHack("got DISPLAY from env: '%s'", display)
 	if display == "" {
 		defaultDisplay := ":0"
-		logRootHack("don't have DISPLAY from env, set to '%s'", defaultDisplay)
+		logHack("don't have DISPLAY from env, set to '%s'", defaultDisplay)
 		err = os.Setenv("DISPLAY", defaultDisplay)
 		if err != nil {
 			fmt.Printf("error setenv DISPLAY: %v\n", err)
 		}
 	}
+
+	iconPath, err := embeds.EmbedIcon()
+	if err != nil {
+		logHack("error: create icon: %v", err)
```

Comment:
I this this is real error so we need to log it with `fmt.Printf` as above, not with `logHack`

Reply:
You also need `\n` at the end, because `Printf` doesn't add it in contrast to `Println`